internal/conf/confdefaults/confdefaults.go

```diff
@@ -93,6 +93,23 @@ var App = conftypes.RawUnified{
 	"experimentalFeatures": {		
 		"structuralSearch": "disabled"
 	},
+	"completions":{
+		"enabled":true,
+		"provider":"sourcegraph"
+	},
+	"embeddings":{
+		"enabled":true,
+		"provider":"sourcegraph",
+		"incremental":true,
+		"excludedFilePathPatterns": [
+			"*.svg",
+			"*.png",
+			"*.jpeg",
+			"*.jpg",
+			"**/__mocks__/**",
+			"**/test/**"
```

Comment:
nitty mc nitpick here

```suggestion
	"completions": {
		"enabled": true,
		"provider": "sourcegraph"
	},
	"embeddings": {
		"enabled": true,
		"provider": "sourcegraph",
		"incremental": true,
		"excludedFilePathPatterns": [
			"*.svg",
			"*.png",
			"*.jpeg",
			"*.jpg",
			"**/__mocks__/**",
			"**/test/**"
```